src/main/java/bit/javaoop/FixedEmployeeNumberManager.java

```diff
@@ -1,27 +1,31 @@
 package bit.javaoop;
 
 
-public class FixedEmployeeNumberMenager extends Menager{
+import java.util.List;
+
+public class FixedEmployeeNumberManager extends Manager {
 
     private int limitOfEmployees;
     private int spotsLeft;
 
-    FixedEmployeeNumberMenager(String name, double salary, int limitOfEmployees) {
-        super(name, salary);
+    public FixedEmployeeNumberManager(String name, double salary, int limitOfEmployees,  List<AbstractEmployee> listOfEmployees) {
+        super(name, salary,listOfEmployees);
```

Comment:
Fajnie teraz użytkownik może zapodać dowolną implementacje, ale jakby czytając ten konstruktor pierwsze co nasuwa się na myśl to: o fajnie mam konstruktor do którego mogę zapodać odrazu listę wypełnioną pracownikami.
Ale żeby troszkę ułatwić dodałbym drugi konstruktor taki: 
![image](https://user-images.githubusercontent.com/18233796/38509802-429faa66-3c23-11e8-948a-7ea96faabcaa.png)
Teraz już w ogóle jest super używając klasy można zapodać implementacje ale nie trzeba możemy użyć konstruktora bez listy z defaultową implementacją i w ogóle o tym nie myśleć.